lib/ex_prompt.ex

```diff
@@ -168,4 +205,18 @@ defmodule ExPrompt do
         pw_loop(prompt)
     end
   end
+
+  defp string_prompt_formatter(prompt, ""), do: prompt
+  defp string_prompt_formatter(prompt, nil), do: prompt
+  defp string_prompt_formatter(prompt, default), do: "(#{default}) #{prompt}"
+
+  defp boolean_prompt_formatter(prompt, nil), do: "#{prompt} [yn] "
+  defp boolean_prompt_formatter(prompt, true), do: "#{prompt} [Yn] "
+  defp boolean_prompt_formatter(prompt, false), do: "#{prompt} [yN] "
+  
+  defp list_prompt_formatter(prompt, _choices, 0), do: "\n#{String.trim(prompt)}\s"
+  defp list_prompt_formatter(prompt, choices, default) do
+    "\n#{String.trim(prompt)} (#{Enum.at(choices, default - 1)})\s"
+  end
+
```

Comment:
I think that naming these without the suffix `_formatter` is fine, WDYT?

Reply:
I agree, it's more readable.